kernel/kernel_drivers.go

```diff
@@ -217,6 +253,33 @@ type KernelDriversTreeOptions struct {
 	KernelInstall bool
 }
 
+// MountPoints describes mount points for a snap or a component.
+type MountPoints struct {
+	// Current is where the container to be installed is currently
+	// available
+	Current string
+	// Target is where the container will be found in a running system
+	Target string
+}
+
+func (mp *MountPoints) UnderCurrentPath(dirs ...string) string {
+	return filepath.Join(append([]string{mp.Current}, dirs...)...)
+}
+
+func (mp *MountPoints) UnderTargetPath(dirs ...string) string {
+	return filepath.Join(append([]string{mp.Target}, dirs...)...)
+}
+
+func (mp *MountPoints) CurrentEqualsTarget() bool {
+	return mp.Current == mp.Target
+}
+
+// ModulesCompInfo contains mount points for a component plus its name.
+type ModulesCompInfo struct {
```

Comment:
my preference would be to also call this ModulesCompMountPoints ?  there's an another Info in this package and is very different, so having both be called Info is confusing to me

Reply:
Changed